### PR TITLE
Refactor: move prism to `@astrojs/prism/components`

### PR DESCRIPTION
**Pull request by bholmesdev**
## Changes

- Adds the `<Prism/>` component as default export from `@astrojs/prism/component`
- Removes the `<Prism/>` component from `astro/components`

☝️ Open to avoiding removal if we prefer. I think adding this component to `@astrojs/prism` could aid in the 1.0 beta move though.

## Testing

N/A

## Docs

https://github.com/withastro/docs/pull/256

### CODE REVIEW

File: packages/astro-prism/components/Prism.astro [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 import Prism from 'prismjs';
-import { addAstro } from '@astrojs/prism';
+import { addAstro } from '../';
```
Comment (matthewp) [inline packages/astro-prism/components/Prism.astro:3]: Can you point to the file here? Helps readability.

File: packages/astro-prism/package.json [hunk 2]
```diff
@@ -13,8 +13,13 @@
   "homepage": "https://astro.build",
   "main": "index.mjs",
   "scripts": {},
+  "files": [
+    "components"
+  ],
   "exports": {
-    ".": "./index.mjs"
+    ".": "./index.mjs",
+    "./components": "./components/index.js",
```
Comment (matthewp) [inline packages/astro-prism/package.json:21]: Nit, but just `@astrojs/prism/component` with no (s)? There's not going to be multiple of these are there?

**Review by matthewp — COMMENTED** [review]
Added some comments

File: packages/astro-prism/package.json (hunk 2 above)
Comment (bholmesdev) [reply packages/astro-prism/package.json]: Update: changed to a default export:

```js
import Prism from '@astrojs/prism/component';
```

Comment (matthewp) [reply packages/astro-prism/package.json]: Yeah i was thinking that from the main would be nice but can't really do that, too bad. This works fine though, thanks!

**Review by matthewp — APPROVED** [review]

**Comment by FredKSchott** [comment]
This is more than enough for now, but whenever we go to expose this to users I think the `/component` entrypoint API is a bit unexpected. I'd expect `import {Prism} from '@astro/prism'` to work.

I know this package is already doing double-duty, But, if the `addAstro` export will always be the "only used internally" export and `Prism` is the publicly available one, then I think a switch of the two would make sense when we go to expose this for users.

```
import {Prism} from '@astrojs/prism';
// in Astro: import {addAstro} from '@astrojs/prism/internal';
```

**Pull request merged**